tests/fuzzing/igl_fuzzer.cpp

```diff
@@ -0,0 +1,34 @@
+// This file is part of libigl, a simple c++ geometry processing library.
+// 
+// Copyright (C) 2021 Alec Jacobson <alecjacobson@gmail.com>
+// 
+// This Source Code Form is subject to the terms of the Mozilla Public License 
+// v. 2.0. If a copy of the MPL was not distributed with this file, You can 
+// obtain one at http://mozilla.org/MPL/2.0/.
+
+
+#include <stdint.h>
+#include <string.h>
+#include <stdlib.h>
+#include <igl/MshLoader.h>
+#include <iostream>
+#include <fstream>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size){
```

Comment:
Why is this linked as `extern "C"`?

Reply:
It is a requirement to expose it to Libfuzzer when we are dealing with C++